### PR TITLE
Ports sign language

### PR DESCRIPTION
**Pull request by Anewbe**
https://github.com/Baystation12/Baystation12/pull/14363

### CODE REVIEW

**Comment by Anewbe** [comment]
Is the fix being worked on presently?

**Comment by Anewbe** [comment]
Fixed that bug.

File: code/game/objects/items/devices/radio/radio.dm [hunk 1]
```diff
@@ -274,6 +274,8 @@ var/global/list/default_medbay_channels = list(
 	//  Fix for permacell radios, but kinda eh about actually fixing them.
 	if(!M || !message) return 0
 
+	if(speaking.flags & NO_TALK_MSG) return 0
```
Comment (GinjaNinja32) [inline code/game/objects/items/devices/radio/radio.dm]: This is *not* the fix. NO_TALK_MSG is *not* for stopping things being *spoken* over radio, it's for stopping the "X talks into their radio" message.

**Comment by Casper3667** [comment]
while you at it may i recommend fix number one and two? https://github.com/Baystation12/Baystation12/pull/14377 and https://github.com/Baystation12/Baystation12/pull/14531

**Comment by Anewbe** [comment]
Oh, nice. Thank you.

**Comment by Casper3667** [comment]
and this for the radio bug: https://github.com/Baystation12/Baystation12/pull/14569

File: code/game/objects/items/devices/radio/radio.dm [hunk 2]
```diff
@@ -274,6 +274,10 @@ var/global/list/default_medbay_channels = list(
 	//  Fix for permacell radios, but kinda eh about actually fixing them.
 	if(!M || !message) return 0
 
+	if(speaking.flags & NONVERBAL) return 0
+
+	if(speaking.flags & SIGNLANG) return 0
```
Comment (GinjaNinja32) [inline code/game/objects/items/devices/radio/radio.dm]: Bug on Bay, just being fixed now; these two need condensing into `if(speaking && (speaking.flags & (SIGNLANG|NONVERBAL)))`, or at the very least a check for `speaking` not being null. Right now AI radio speech runtimes if they don't have a default language set.

Comment (GinjaNinja32) [inline code/game/objects/items/devices/radio/radio.dm]: Baystation12/Baystation12#14571

**Pull request merged**